app/resources/app-base.html.template

```diff
@@ -15,13 +15,14 @@
 {% block logo %}
   {% if env.repo and env.action and env.action != "start" %}
     {% if env.rtl %}
-	  <button class="title-bar-button"><a href="{{env.start_url}}"><img src="{{env.global_url}}/forward_arrow.png" alt=""></a></button>Google Person Finder
+      <a href="{{env.start_url}}"><img class="title-bar-button" src="{{env.global_url}}/forward_arrow.png" alt="Forward"></a>
```

Comment:
Alt attribute for this should be still "Back" because it's a button to go back (even though it's right arrow).